fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -36,8 +38,10 @@ def run!
         OptionParser::MissingArgument => e
         abort e.to_s
       rescue FastlaneCore::Interface::FastlaneError => e # user_error!
+        collector.did_raise_error(@program[:name])
         display_user_error!(e, e.message)
       rescue => e # high chance this is actually FastlaneCore::Interface::FastlaneCrash, but can be anything else
+        collector.did_raise_error(@program[:name])
```

Comment:
Hm, I think this may actually be "wrong" (we're passing a string, but expecting a symbol) but actually work. We're not picking up on it in any tests.